backend/app/Migration.hs

```diff
@@ -32,16 +33,27 @@ import Validation (ReportError (..), validateReport)
 banList :: PlayerBanList
 banList = ["mordak", "mellowsedge"]
 
+insertLegacyEntry :: (MonadIO m, MonadLogger m) => ParsedLegacyLadderEntry -> DBAction m ()
```

Comment:
I moved this from the Database module to avoid circular dependencies. It hasn't changed though.